styles/styles.css

```diff
@@ -227,23 +284,196 @@ hr {
 }
 
 .footer {
-  height: 625px;
   background-color: #10367c;
-  display: flex;
-  flex-direction: column;
-  justify-content: space-between;
 }
 
-.footer-inf {
-  display: flex;
-  gap: 16px;
+.footer__logo--responsive {
+  display: none;
 }
 
-.footer-inf__block {
-  width: 400px;
+.footer-inf {
+  margin-bottom: 64px;
 }
 
 .footer-inf__copyright {
   font-size: 14px;
   color: #ffffff;
 }
+
+/* For tablet: */
+@media (max-width: 1024px) {
+  .col-md-1 {
+    width: var(--col-width);
+  }
+
+  .col-md-2 {
+    width: calc(var(--col-width) * 2);
+  }
+
+  .col-md-3 {
+    width: calc(var(--col-width) * 3);
+  }
+
+  .col-md-4 {
+    width: calc(var(--col-width) * 4);
+  }
+
+  .col-md-5 {
+    width: calc(var(--col-width) * 5);
+  }
+
+  .col-md-6 {
+    width: calc(var(--col-width) * 6);
+  }
+
+  .col-md-7 {
+    width: calc(var(--col-width) * 7);
+  }
+
+  .col-md-8 {
+    width: calc(var(--col-width) * 8);
+  }
+
+  .col-md-9 {
+    width: calc(var(--col-width) * 9);
+  }
+
+  .col-md-10 {
+    width: calc(var(--col-width) * 10);
+  }
+
+  .col-md-11 {
+    width: calc(var(--col-width) * 11);
+  }
+
+  .col-md-12 {
+    width: calc(var(--col-width) * 12);
+  }
+
+  .container {
+    padding: 80px 16px;
+  }
+
+  .offer__item {
+    margin-bottom: 40px;
+  }
+
+  .offer__items {
+    margin-bottom: -40px;
+  }
+}
+
+/* For mobile: */
+@media (max-width: 768px) {
+  :root {
+    --col-width: calc(100% / 6);
+  }
+
+  .col-sm-1 {
+    width: var(--col-width);
+  }
+
+  .col-sm-2 {
+    width: calc(var(--col-width) * 2);
+  }
+
+  .col-sm-3 {
+    width: calc(var(--col-width) * 3);
+  }
+
+  .col-sm-4 {
+    width: calc(var(--col-width) * 4);
+  }
+
+  .col-sm-5 {
+    width: calc(var(--col-width) * 5);
+  }
+
+  .col-sm-6 {
+    width: calc(var(--col-width) * 6);
+  }
+
+  .container {
+    padding: 40px 16px;
+  }
+
+  hr {
+    display: none;
+  }
+
+  .basic-text {
+    font-size: 15px;
+    line-height: 18px;
+  }
+
+  .title {
+    margin-bottom: 32px;
+    font-size: 28px;
+    line-height: 33px;
+  }
+
+  .offer__container {
+    padding: 24px 73px 40px;
```

Comment:
It would be better to replace this padding with an accrual that is a multiple of 5